src/components/grids.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import styled from 'styled-components'
+
+const gapSize = 20
+const numberOfColumns = 12
+
+export const Row = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  flex: 0 1 auto;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-right: -${gapSize / 2}px;
+  margin-left: -${gapSize / 2}px;
+  margin-bottom: ${gapSize}px;
+`
+
+export const Col = styled.div`
+  box-sizing: border-box;
+  flex: 0 0 auto;
+  padding-right: ${gapSize / 2}px;
+  padding-left: ${gapSize / 2}px;
+  flex-basis: ${props => (100 / numberOfColumns) * (props.size)}%;
+  max-width: ${props => (100 / numberOfColumns) * (props.size)}%;
```

Comment:
> It seems like the value of `flex-grow` is relative to other columns while the current grid system has the fixed scale of 12 in total.

I’m thinking that maybe less static numbers of columns is good for responsive pages (now that Force and microgravity have merged) as well as indeed share with React Native?

> I feel like we should experiment with `flex` or other flex-related properties before we get too far so we can make grid perfectly re-usable across multiple platforms.

I don’t think we need to engineer it too much right now, I think we should aim for the simplest solution that works well for what we need _right now_, so I’d suggest you take the time to make the best use of flexbox (and e.g. get rid of manual calculations).

> What are the browser versions we are targeting?

Good question. @craigspaeth can you say what historically has been the target?

> Also, is there any example iOS app that actually uses reaction-force that we can use for acceptance?

Nah, not yet. Don’t worry right now about these components working or not for iOS. We’ll cross that bridge when we come to it, just make sure to keep it simple so that it will be flexible to adjust to when we _do_ try to make it cross platform.